src/pages/home/ReportScreen.js

```diff
@@ -239,81 +236,91 @@ class ReportScreen extends React.Component {
                 style={[styles.appContent, styles.flex1, {marginTop: this.state.viewportOffsetTop}]}
                 keyboardAvoidingViewBehavior={Platform.OS === 'android' ? '' : 'padding'}
             >
-                <OfflineWithFeedback
-                    pendingAction={addWorkspaceRoomPendingAction}
-                    errors={addWorkspaceRoomErrors}
-                    errorRowStyles={styles.dNone}
-                >
-                    <HeaderView
-                        reportID={reportID}
-                        onNavigationMenuButtonClicked={() => Navigation.navigate(ROUTES.HOME)}
-                    />
-                </OfflineWithFeedback>
-                {this.props.accountManagerReportID && ReportUtils.isConciergeChatReport(this.props.report) && this.state.isBannerVisible && (
-                    <Banner
-                        containerStyles={[styles.mh4, styles.mt4, styles.p4, styles.bgDark]}
-                        textStyles={[styles.colorReversed]}
-                        text={this.props.translate('reportActionsView.chatWithAccountManager')}
-                        onClose={this.dismissBanner}
-                        onPress={this.chatWithAccountManager}
-                        shouldShowCloseButton
-                    />
-                )}
-                <View
-                    nativeID={CONST.REPORT.DROP_NATIVE_ID}
-                    style={[styles.flex1, styles.justifyContentEnd, styles.overflowHidden]}
-                    onLayout={event => this.setState({skeletonViewContainerHeight: event.nativeEvent.layout.height})}
+                <FullPageNotFoundView
+                    shouldShow={!this.props.report.reportID}
```

Comment:
If we are viewing a report and it gets deleted we will get the not found view, this caused https://github.com/Expensify/App/issues/28969. We should have taken into consideration if the report was previously accessible.